css/index.css

```diff
@@ -1,19 +1,75 @@
 /* Header */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
```

Comment:
This code block is not for header.
Add comments to describe what this block does.